tasks/scripts/check-distribution-env/diff

```diff
@@ -28,6 +28,7 @@ main() {
     exit 0
   fi
 
+	echo "Error: found some variables in BOSH packaging that might not be supported by the Concourse binary:"
```

Comment:
Since this script is generic across bosh and helm, I think
```suggestion
	echo "Error: found some variables in ${DISTRIBUTION} packaging that might not be supported by the Concourse binary:"
```
would be better here.

Reply:
makes sense!